src/libraries/System.Formats.Tar/tests/TarReader/TarReader.TarEntry.ExtractToFile.Tests.Unix.cs

```diff
@@ -10,36 +10,35 @@ namespace System.Formats.Tar.Tests
     public partial class TarReader_TarEntry_ExtractToFile_Tests : TarTestsBase
     {
         [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.tvOS)] // https://github.com/dotnet/runtime/issues/68360
-        [Fact]
-        public async Task SpecialFile_Unelevated_Throws_Async()
+        [ConditionalFact(nameof(IsUnixButNotSuperUser))]
+        public void SpecialFile_Unelevated_Throws()
```

Comment:
I noticed this test, which was supposed to be the synchronous one, was written as the asynchronous one.